iothub/device/src/Transport/RetryDelegatingHandler.cs

```diff
@@ -735,7 +736,17 @@ private async Task EnsureOpenedAsync(CancellationToken cancellationToken)
                         Logging.Info(this, "Opening connection", nameof(EnsureOpenedAsync));
```

Comment:
you can remove the `if (Logging.IsEnabled)` check :D

Reply:
across the whole file

Reply:
hahaha, good point! I'll do that